api/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM public.ecr.aws/lambda/python:3.8 AS lambda-image
```

Comment:
Should we use a new version of Python, like 31.3

Reply:
I don't want to change this too much and this image is only used to pull some libraries out to allow the lambda to run. The python bit isn't really needed.